test/check-application

```diff
@@ -51,13 +51,88 @@ class TestApplication(testlib.MachineCase):
         # show container listing toggle
         b.click('#containers-containers tbody tr:contains("busybox:latest") td.listing-ct-toggle')
         b.wait_present('#containers-containers tbody tr:contains("busybox:latest") + tr button.btn-delete')
-
+        #show running container
         b.click("#containers-containers-filter button")
         b.wait_present("#containers-containers-filter .dropdown-menu")
         b.click("#containers-containers-filter li[data-data='running'] a")
+        b.wait_present('#containers-containers tr:contains("busybox:latest")')
+        b.wait_visible('#containers-containers tr:contains("busybox:latest") + tr:contains("sleep 1000")')
+
+        # switch to all containers
+        b.wait_present("#containers-containers-filter button")
+        b.click("#containers-containers-filter button")
+        b.wait_visible("#containers-containers-filter .dropdown-menu")
+        b.click("#containers-containers-filter li[data-data='all'] a")
+        b.wait_present("#containers-containers")
+        b.wait_present('#containers-containers tr:contains("busybox:latest") td:contains("running")')
+        b.wait_present('#containers-containers tr:contains("alpine:latest")')
```

Comment:
This is now actually doing some of the above, but there are now two sections that switch to "all" containers. Please only have one of them (looks like you can just delete the one above).

Reply:
I want to delete a stoped container under all containers and to check it has been deleted, not shown under all container list. So I switched to all container here.

Reply:
Yes, that makes sense. But there is a similar section of code above which now does nothing, that's the one to get rid of.

Reply:
And show all container above is test for the dropdown filter.